src/status_im/ui/screens/profile/components/sheets.cljs

```diff
@@ -0,0 +1,44 @@
+(ns status-im.ui.screens.profile.components.sheets
+  (:require-macros [status-im.utils.views :refer [defview letsubs] :as views])
```

Comment:
don't refer even these macros, use the `views/` ns alias. I known some ns don't do it but that's old code or mistakes